core/lib/zksync_core/src/api_server/web3/pubsub.rs

```diff
@@ -49,18 +55,14 @@ impl<V: Clone> PubSubNotifier<V> {
             .context("get_sealed_miniblock_number()")
     }
 
-    async fn current_subscribers(&self) -> Vec<V> {
-        self.subscribers.read().await.values().cloned().collect()
-    }
```

Comment:
Can no longer do that, it clones and returns the subscription structures and these clones would alter be dropped causing the subscription to close.